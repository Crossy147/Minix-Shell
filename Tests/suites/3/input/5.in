# simple pipes

ls servers/pm > test.e4.3.A

cat test.e4.3.A test.e4.3.A test.e4.3.A | sort 

echo ------------------------
cat test.e4.3.A | sort | wc
echo ------------------------
cat test.e4.3.A | cat | cat | sort | cat | cat | cat | cat | cat | cat > test.e4.3.B

cat  test.e4.3.B | cat | cat |cat | cat | cat |cat | cat | cat |cat > test.e4.3.C
cat test.e4.3.C | cat | wc

rm test.e4.3.A test.e4.3.B test.e4.3.C
