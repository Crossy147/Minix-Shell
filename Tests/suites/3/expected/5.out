Makefile
Makefile
Makefile
alloc.c
alloc.c
alloc.c
break.c
break.c
break.c
const.h
const.h
const.h
exec.c
exec.c
exec.c
forkexit.c
forkexit.c
forkexit.c
getset.c
getset.c
getset.c
glo.h
glo.h
glo.h
main.c
main.c
main.c
misc.c
misc.c
misc.c
mproc.h
mproc.h
mproc.h
param.h
param.h
param.h
pm.h
pm.h
pm.h
proto.h
proto.h
proto.h
signal.c
signal.c
signal.c
table.c
table.c
table.c
time.c
time.c
time.c
timers.c
timers.c
timers.c
trace.c
trace.c
trace.c
type.h
type.h
type.h
utility.c
utility.c
utility.c
------------------------
     21     21    167 
------------------------
     21     21    167 
